Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
@vhbit line 778 emits completion immediately after next. Meaning if it was a streamed response you only get a single emission regardless and dispose after it. **if** streamed responses were supported there that’s how it could possibly break it.

But as I said I think we should probably not worry about it.